src/csharpExtensionExports.ts

```diff
@@ -20,4 +22,10 @@ export interface CSharpExtensionExports {
     initializationFinished: () => Promise<void>;
     logDirectory: string;
     profferBrokeredServices: (container: GlobalBrokeredServiceContainer) => void;
+    sendRequest: <Params, Response, Error>(
```

Comment:
I could certainly do this, perhaps have an `experimental` object that is the thing that defines these methods?

Reply:
An experimental object seems reasonable. Although this method doesn't worry me too much since I think this is something we'd have long term in it's current form; I'd absolutely expect the server path function to go away though once we create a proper way to contribute stuff since we obviously shouldn't be copying DLLs long term.